frontend/packages/kubevirt-plugin/src/k8s/requests/vm/actions.ts

```diff
@@ -49,7 +49,7 @@ const VMActionRequest = async (
         name: getName(vm),
         namespace: getNamespace(vm),
         body: JSON.stringify(body),
-      })
+      } as RequestInit)
```

Comment:
you shouldn't need to force the `RequestInit` type here. Looks like `name` and `namespace` props are used incorrectly.

Reply:
Updated.